app/lib/medical-transcription-analysis-stack.ts

```diff
@@ -431,11 +431,11 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
     
           methods.forEach(method => {
             apiResource.addMethod(method, undefined, {
-              authorizationType: apigateway.AuthorizationType.COGNITO,
-              authorizer: {
-                authorizerId: `${authorizer.ref}`
-                //`${theauthorizer.ref}`
-              }
+              authorizationType: apigateway.AuthorizationType.NONE//apigateway.AuthorizationType.COGNITO,
+              // authorizer: {
+              //   authorizerId: `${authorizer.ref}`
```

Comment:
As discussed, work on adding the authorizer will be integrated in coming stages (but before we merge into master)